libyang/schema.py

```diff
@@ -878,6 +901,68 @@ def __str__(self):
         return self.name()
 
 
+# -------------------------------------------------------------------------------------
+class Identity:
```

Comment:
It would be nicer if we could use the register mechanism like for the other node types. Is there a problem with that?

Reply:
Identity is not SNode/PNode based. Even the underlying C structure those are separated.